static/js/around-map.js

```diff
@@ -108,8 +113,9 @@ function findPoi (id) {
 function highlight (poi_id) {
   const cardActive = document.querySelector('.current-card')
   const markerActive = document.querySelector('.current-marker')
-  const card = document.querySelector('#card-'+poi_id)
-  const marker = document.querySelector('#'+poi_id)
+  const card = document.querySelector(`#card-${poi_id}`)
+  const marker = document.getElementById(poi_id)
```

Comment:
why are you looking with `.querySelector('#…` and then with `.getElementById(…`?
Just take one way for consistency, or if you have a hackish reason, write an comment explaining *why* you must do that.